source/help/getting-started/searching.rst

```diff
@@ -120,4 +120,4 @@ MySQL:
 
 .. _documentation: https://docs.mattermost.com/install/i18n.html
 .. _MySQL: http://dev.mysql.com/doc/refman/5.7/en/fulltext-stopwords.html
-.. _Postgres: http://apt-browse.org/browse/ubuntu/precise/main/i386/postgresql-9.1/9.1.3-2/file/usr/share/postgresql/9.1/tsearch_data/english.stop
+.. _Postgres: https://www.apt-browse.org/browse/ubuntu/precise/main/amd64/postgresql-9.1/9.1.3-2/file/usr/share/postgresql/9.1/tsearch_data/english.stop
```

Comment:
nit: The link looks good, but as per our [software requirements](https://docs.mattermost.com/install/requirements.html#database-software) we support postgres 9.4+, so I'd suggest to change it to point to the updated 9.4 package file (which has the exact same contents as the one linked right now) only for consistency with the version in the docs.

```suggestion
.. _Postgres: https://www.apt-browse.org/browse/debian/jessie/main/amd64/postgresql-9.4/9.4.18-0+deb8u1/file/usr/share/postgresql/9.4/tsearch_data/english.stop
```